closed/adds/jdk/src/share/native/jdk/crypto/jniprovider/NativeCrypto.c

```diff
@@ -153,6 +181,7 @@ static void printErrors(void) {
     fflush(stderr);
 }
 
+static void *handle = NULL;
```

Comment:
Now this is not local to `loadCrypto()` it could do with a better, more descriptive name.